envs/doom/multiplayer/doom_multiagent.py

```diff
@@ -120,9 +124,13 @@ def _ensure_initialized(self):
                 self.game.add_game_args(f'-record {demo_path}')
         else:
             # Join existing game.
+            try:
+                vizdoom_env_timeout = int(os.environ['TRAVIS_VIZDOOM_ENV_TIMEOUT'])
```

Comment:
I think this code will be better if you query the environment variable outside of if.
Is there a reason for this code to be duplicated? Usually it is always better to avoid code duplication, DRY https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#DRY_vs_WET_solutions